cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -414,6 +414,14 @@
   </property>
 
   <property>
+    <name>apps.update.schedules</name>
+    <value>true</value>
+    <description>
+      Enable if changes in schedules of an application can be updated on app redeploy.
```

Comment:
This property does not affect update from schedule endpoint. It will be good to explicitly say this here